components/Learn/PreviousNextLink/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Link from 'next/link';
+import { FaAngleDoubleLeft, FaAngleDoubleRight } from 'react-icons/fa';
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import type { FC } from 'react';
+import type { LinkInfo } from '../../../types';
+
+type PreviousNextLinkProps = {
+  previous?: LinkInfo;
+  next?: LinkInfo;
+};
+
+const PrevNextLink: FC<PreviousNextLinkProps> = ({ previous, next }) => {
```

Comment:
Can we rename the component to be `PreviousNextLink` to respect the folder naming.